src/components/Calculator.js

```diff
@@ -1,15 +1,117 @@
+import calculate from '../logic/calculate';
 import '../styles.css';
 
-function Calculator() {
+const Calculator = () => {
+  const [count, setCount] = useState({
+    total: 0,
+    operation: null,
+    next: null,
+  });
+
+  const clickHandler = (button) => {
+    const output = calculate(count, button);
+    setCount(output);
+  };
+
   return (
     <main className="container">
       <section className="calculator">
-        <Button />
+        <header className="header calc">
+          {count.total}
+          {' '}
+          {count.operation}
+          {' '}
+          {count.next}
+        </header>
+        <button
+          className="btn"
+          type="button"
+          onClick={() => clickHandler('AC')}
+        >
+          Ac
+        </button>
+        <button type="button" onClick={() => clickHandler(' +/-')}>
+          +/-
+        </button>
+        <button type="button" onClick={() => clickHandler('%')}>
+          %
+        </button>
+        <button
+          type="button"
+          className="color"
+          onClick={() => clickHandler('÷')}
+        >
+          ÷
+        </button>
+        <button type="button" onClick={() => clickHandler('7')}>
+          7
+        </button>
+        <button type="button" onClick={() => clickHandler('8')}>
+          8
+        </button>
+        <button type="button" onClick={() => clickHandler('9')}>
+          9
+        </button>
+        <button
+          type="button"
+          className="color"
+          onClick={() => clickHandler('x')}
+        >
+          x
+        </button>
+        <button type="button" onClick={() => clickHandler('4')}>
+          4
+        </button>
+        <button type="button" onClick={() => clickHandler('5')}>
+          5
+        </button>
+        <button type="button" onClick={() => clickHandler('6')}>
+          6
+        </button>
+        <button
+          type="button"
+          className="color"
+          onClick={() => clickHandler('-')}
+        >
+          -
+        </button>
+        <button type="button" onClick={() => clickHandler('1')}>
+          1
+        </button>
+        <button type="button" onClick={() => clickHandler('2')}>
+          2
+        </button>
+        <button type="button" onClick={() => clickHandler('3')}>
+          3
+        </button>
+        <button
+          type="button"
+          className="color"
+          onClick={() => clickHandler('+')}
+        >
+          +
+        </button>
+        <button
+          type="button"
+          className="zero"
+          onClick={() => clickHandler('0')}
+        >
+          0
+        </button>
+        <button type="button" onClick={() => clickHandler('.')}>
+          .
+        </button>
+        <button
+          className="btn color"
+          type="button"
+          onClick={() => clickHandler('=')}
+        >
+          =
+        </button>
       </section>
     </main>
   );
-}
+};
```

Comment:
- [ ] Great work so far 👍 but I guess you misunderstood a requirement which was to practice state and props and you are using state but not props 🤔 Kindly make a component that can be used in the Calculator.js component and you can pass props to the child component to follow the project requirements 😊